gbpservice/nfp/modules/configurator.py

```diff
@@ -0,0 +1,458 @@
+        invoked by the configurator REST server. It updates configuration
+        of a network service based on the configuration request specified
+        in the request_data argument.
+
+        :param context: RPC context instance
+        :param request_data: RPC data
+
+        Returns: None
+
+        """
+
+        try:
+            self._send_request('update', request_data)
+        except Exception as err:
+            msg = ("Failed to update network device configuration. %s" %
+                   str(err).capitalize())
+            LOG.error(msg)
+
+    def create_network_service_config(self, context, request_data):
+        """RPC method to configure a network service.
+
+        Configures a network service specified in the request data. This
+        RPC method is invoked by the configurator REST server. It configures
+        a network service based on the configuration request specified in
+        the request_data argument.
+
+        :param context: RPC context instance
+        :param request_data: RPC data
+
+        Returns: None
+
+        """
+
+        try:
+            self._send_request('create', request_data)
+        except Exception as err:
+            msg = ("Failed to create network service configuration. %s" %
+                   str(err).capitalize())
+            LOG.error(msg)
+
+    def delete_network_service_config(self, context, request_data):
+        """RPC method to clear configuration of a network service.
+
+        Clears configuration of a network service. This RPC method is
+        invoked by the configurator REST server. It clears configuration
+        of a network service based on the configuration request specified
+        in the request_data argument.
+
+        :param context: RPC context instance
+        :param request_data: RPC data
+
+        Returns: None
+
+        """
+
+        try:
+            self._send_request('delete', request_data)
+        except Exception as err:
+            msg = ("Failed to delete network service configuration. %s" %
+                   str(err).capitalize())
+            LOG.error(msg)
+
+    def update_network_service_config(self, context, request_data):
+        """RPC method to update of configuration in a network service.
+
+        Updates configuration of a network service. This RPC method is
+        invoked by the configurator REST server. It updates configuration
+        of a network service based on the configuration request specified
+        in the request_data argument.
+
+        :param context: RPC context instance
+        :param request_data: RPC data
+
+        Returns: None
+
+        """
+
+        try:
+            self._send_request('update', request_data)
+        except Exception as err:
+            msg = ("Failed to update network service configuration. %s" %
+                   str(err).capitalize())
+            LOG.error(msg)
+
+    def get_notifications(self, context):
```

Comment:
Yes. get_notifications() is an RPC call and all others are RPC cast. This API returns list of notifications.